src/input.js

```diff
@@ -67,12 +65,6 @@ class Input extends Component {
     this.updateInputValue(event.target.value)
   }
 
-  handleBlur = () => {
-    if (!this.autocomplete.isMouseDown) {
-      this.autocomplete.reset()
-    }
-  }
-
   reset = () => {
     moveCursorToTheEnd(this._inputNode)
```

Comment:
@psicotropicos I think this was also supposed to be removed. 

I tried this PR in a project and wasn't unable to unfocus in Safari.